build-system/tasks/dist.js

```diff
@@ -55,6 +60,37 @@ const {maybeUpdatePackages} = require('./update-packages');
 const {green, cyan} = colors;
 const argv = require('minimist')(process.argv.slice(2));
 
+const babel = require('@babel/core');
```

Comment:
We use `browserify` + `babelify` everywhere else in `build-system`. For uniformity, any reason not to do so here as well? For example, single pass does this:

https://github.com/ampproject/amphtml/blob/4b86736870dc948e0c706fcac6317136d16af819/build-system/compile/single-pass.js#L315-L328

Reply:
single pass only needs browserify for the dependency graph discovery. the babel core usage is in line to https://github.com/ampproject/amphtml/blob/master/build-system/compile/single-pass.js#L475-L482

also under the hood i would expect babelify to be using `@babel/core`